packages/cms-cli/commands/logs.js

```diff
@@ -171,20 +169,26 @@ exports.builder = yargs => {
     describe: 'Path to serverless function',
     type: 'string',
   });
-  yargs.option('latest', {
-    alias: 'l',
-    describe: 'retrieve most recent log only',
-    type: 'boolean',
-  });
-  yargs.option('compact', {
-    alias: 'c',
-    describe: 'output compact logs',
-    type: 'boolean',
-  });
-  yargs.option('follow', {
-    alias: ['t', 'tail', 'f'],
-    describe: 'follow logs',
-    type: 'boolean',
+  yargs.options({
+    latest: {
+      alias: 'l',
+      describe: 'retrieve most recent log only',
+      type: 'boolean',
+    },
+    compact: {
+      describe: 'output compact logs',
+      type: 'boolean',
+    },
+    follow: {
+      alias: ['t', 'tail', 'f'],
+      describe: 'follow logs',
+      type: 'boolean',
+    },
+    limit: {
+      alias: ['limit', 'n', 'max-count'],
+      describe: 'limit the number of logs to output',
```

Comment:
Does this interact with --follow?

Reply:
Good catch. I added `.conflict('follow', 'limit')`so they cannot be used together.